scripts/seed.py

```diff
@@ -100,49 +116,85 @@ def create_challenge_host_team(user):
     """
+    anon_counter = 0  # two private leaderboards
     for i in xrange(number_of_challenges):
-        if (i % 3 == 0):
-            create_challenge("{} Challenge".format(fake.first_name()),
-                             timezone.now() - timedelta(days=100),
-                             timezone.now() + timedelta(days=500),
-                             host_team
-                             )
-        elif (i % 3 == 1):
-            create_challenge("{} Challenge".format(fake.first_name()),
-                             timezone.now() - timedelta(days=500),
-                             timezone.now() - timedelta(days=100),
-                             host_team
-                             )
-        elif (i % 3 == 2):
-            create_challenge("{} Challenge".format(fake.first_name()),
-                             timezone.now() + timedelta(days=100),
-                             timezone.now() + timedelta(days=500),
-                             host_team
-                             )
-
-
-def create_challenge(title, start_date, end_date, host_team):
+
+        is_leaderboard_anon = False
+        if anon_counter < 2:
+            is_leaderboard_anon = True
+            anon_counter += 1
+        else:
+            is_leaderboard_anon = False
+
+        if i % 3 == 0:
+            create_challenge(
+                "{} Challenge".format(fake.first_name()),
+                timezone.now() - timedelta(days=100),
+                timezone.now() + timedelta(days=500),
+                host_team,
+                is_leaderboard_anon,
+            )
+        elif i % 3 == 1:
+            create_challenge(
+                "{} Challenge".format(fake.first_name()),
+                timezone.now() - timedelta(days=500),
+                timezone.now() - timedelta(days=100),
+                host_team,
+                is_leaderboard_anon,
+            )
+        elif i % 3 == 2:
+            create_challenge(
+                "{} Challenge".format(fake.first_name()),
+                timezone.now() + timedelta(days=100),
+                timezone.now() + timedelta(days=500),
+                host_team,
+                is_leaderboard_anon,
+            )
+
+
+def create_challenge(title, start_date, end_date, host_team, anon_leaderboard):
     """
     Creates a challenge.
     """
     evaluation_script = open(
-        os.path.join(settings.BASE_DIR, 'examples', 'example1', 'sample_evaluation_script.zip'), 'rb')
-    queue = ''.join(random.choice(string.ascii_letters) for _ in range(75))
+        os.path.join(
+            settings.BASE_DIR,
+            "examples",
+            "example1",
+            "sample_evaluation_script.zip",
+        ),
+        "rb",
+    )
+    queue = "".join(random.choice(string.ascii_letters) for _ in range(75))
     year = datetime.date.today().year
-    slug = '{t}-{y}'.format(t=title, y=year)
+    # add UUID here
+    uuid_stamp = uuid.uuid4().hex[0:5]
```

Comment:
`uuid_stamp = uuid.uuid4().hex[0:5]` is generally leading to conflicts in uuid_stamp. Can you please take the first 10 chars instead of 5? Tested and it seems to work with 10.